src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/ContentDbContextModelSnapshot.cs

```diff
@@ -1447,51 +1456,70 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         new
                         {
                             Id = new Guid("4fa4fe8e-9a15-46bb-823f-49bf8e0cdec5"),
+                            Created = new DateTime(1, 1, 1, 0, 0, 0, 0, DateTimeKind.Unspecified),
+                            CreatedById = new Guid("00000000-0000-0000-0000-000000000000"),
                             NextReleaseDate = "{\"Year\":\"2019\",\"Month\":\"3\",\"Day\":\"22\"}",
+                            OriginalId = new Guid("00000000-0000-0000-0000-000000000000"),
                             PublicationId = new Guid("cbbd299f-8297-44bc-92ac-558bcf51f8ad"),
                             Published = new DateTime(2018, 3, 22, 0, 0, 0, 0, DateTimeKind.Unspecified),
                             ReleaseName = "2016",
                             Slug = "2016-17",
                             Status = "Draft",
                             TimePeriodCoverage = "AY",
-                            TypeId = new Guid("9d333457-9132-4e55-ae78-c55cb3673d7c")
+                            TypeId = new Guid("9d333457-9132-4e55-ae78-c55cb3673d7c"),
+                            Version = 0
                         },
                         new
                         {
                             Id = new Guid("e7774a74-1f62-4b76-b9b5-84f14dac7278"),
+                            Created = new DateTime(1, 1, 1, 0, 0, 0, 0, DateTimeKind.Unspecified),
+                            CreatedById = new Guid("00000000-0000-0000-0000-000000000000"),
                             NextReleaseDate = "{\"Year\":\"2019\",\"Month\":\"7\",\"Day\":\"19\"}",
+                            OriginalId = new Guid("00000000-0000-0000-0000-000000000000"),
                             PublicationId = new Guid("bf2b4284-6b84-46b0-aaaa-a2e0a23be2a9"),
                             Published = new DateTime(2018, 7, 19, 0, 0, 0, 0, DateTimeKind.Unspecified),
                             RelatedInformation = "[{\"Id\":\"f3c67bc9-6132-496e-a848-c39dfcd16f49\",\"Description\":\"Additional guidance\",\"Url\":\"http://example.com\"},{\"Id\":\"45acb50c-8b21-46b4-989f-36f4b0ee37fb\",\"Description\":\"Statistics guide\",\"Url\":\"http://example.com\"}]",
                             ReleaseName = "2016",
                             Slug = "2016-17",
                             Status = "Draft",
                             TimePeriodCoverage = "AY",
-                            TypeId = new Guid("9d333457-9132-4e55-ae78-c55cb3673d7c")
+                            TypeId = new Guid("9d333457-9132-4e55-ae78-c55cb3673d7c"),
+                            Version = 0
                         },
                         new
                         {
                             Id = new Guid("63227211-7cb3-408c-b5c2-40d3d7cb2717"),
+                            Created = new DateTime(1, 1, 1, 0, 0, 0, 0, DateTimeKind.Unspecified),
+                            CreatedById = new Guid("00000000-0000-0000-0000-000000000000"),
```

Comment:
Is there any superuser id that can be used in the ContentDbContext seed Release data for the CreatedById field?

Reply:
Yup will set appropriate values in ContentDbContext, thanks!

Reply:
Added new migration to update these